spec/system/user_articles_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe "User views their articles" do
+  it "displays a list of articles created by the user" do
+    user = create(:user)
+    visit root_path(as: user)
```

Comment:
**todo:** Drop this visit.

There is no need for this visit. All it does is add an unnecessary page visit to your tests and slow them down. You can drop it and skip that step.